client/app/scripts/liveblog-bloglist/styles/liveblog-bloglist.less

```diff
@@ -138,54 +138,54 @@
     margin-top: 50px;
 }
 .team_settings {
-    .search-result {
-        background: #fff none repeat scroll 0 0;
-        border: 1px solid #ddd;
-        bottom: 80px;
-        left: 40px;
-        overflow: auto;
-        padding: 10px;
-        position: absolute;
-        right: 40px;
-        top: 101px;
-        z-index: 1;
-    }
-    .member-list ul > li {
-        border: 1px solid transparent;
-        border-radius: 3px;
-        float: left;
-        margin: 5px;
-        padding: 15px 10px;
-        position: relative;
-        width: 115px;
-    }
-    .member-list ul > li .name {
-        text-align: center;
-        width: 115px;
-        white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
-    }
-    .member-list ul>li:hover {
-        border-color: rgba(0,0,0,.12);
-        box-shadow: 0 1px 3px 0 rgba(0,0,0,.3);
-        .icon-close-small {
-            display: inline-block;
-        }
-    }
-    .member-list ul > li .avatar {
-        display: block;
-        width: 84px;
-        height: 84px;
-        float: none;
-        margin: 0 auto 10px;
-    }
-    .member-list ul > li i {
-        position: absolute;
-        right: 10px;
-        top: 10px;
-        display: none;
-        opacity: .6;
-        filter: alpha(opacity=60);
-    }
+	.search-result {
```

Comment:
On this project we indent with 4 spaces everywhere (less, js, python...).
Here you indented with tabulation